server/hazard-service/src/main/java/edu/illinois/ncsa/incore/service/hazard/models/hurricane/utils/HurricaneUtil.java

```diff
@@ -27,9 +27,51 @@ public class HurricaneUtil {
     // Time
     private static final String units_hr = "hr";
     private static final String units_min = "min";
+    private static final String units_s = "s";
+
+    // Velocity
+    private static final String units_m_per_s = "m/s";
+    private static final String units_m_per_sec = "m/sec";
+    private static final String units_in_per_s = "in/s";
+    private static final String units_ft_per_s = "ft/s";
+    private static final String units_mph = "mph";
+
+    // Degree
+    private static final String degree = "deg";
+    private static final String radian = "rad";
 
     public static final List<String> distanceUnits = Arrays.asList(units_cm, units_m, units_in, units_ft);
-    public static final List<String> durationUnits = Arrays.asList(units_hr, units_min);
+    public static final List<String> durationUnits = Arrays.asList(units_hr, units_min, units_s);
+    public static final List<String> speedUnits = Arrays.asList(units_m_per_s, units_in_per_s, units_ft_per_s, units_mph);
```

Comment:
I think you need to add units_m_per_sec here.

Reply:
let me know when this code get updated. I will deploy to tst

Reply:
sorry i missed that!